Configuration.mk

```diff
@@ -27,6 +27,10 @@ RANLIB := -ranlib
 READELF := -readelf
 SIZE := -size
 
+# Library versions.
+NEWLIB_VERSION ?= 4.2.0.20211231
+LIBCPP_VERSION ?= 12.2.0
```

Comment:
Should we split this bump out as well

Reply:
I think we might be able to split it up. We may also not want to do more work and review more PRs. I'm still trying to get it to work at all.